### ISSUE TITLE
Virtual Market Seeding

### ISSUE THREAD

**Issue by jdhirst** [EvEmu-Project/evemu_Crucible#138]
This is a stub issue for 0.8.5 to do the following:

- Get rid of actual market seeding. This is super wasteful in resources and is very slow.
- If an administrator enables it, the market can be 'seeded' virtually, which will not actually create sell orders but create them on the fly when someone checks the market, and then only stores the differences in the database.

This will be a bit messy to implement but will give a huge performance boost for both the server and client.

**Comment by zhyrohaad** [comment]
> This will be a bit messy to implement but will give a huge performance boost for both the server and client.

the initial query will affect the entire region.   i would advise against this just on the performance standpoint.
personally, and in practice, it would be a better option to delay the server hardware while seeding markets before the game server is started to provide a more-responsive server, even if it's only once/region.

**Comment by jdhirst** [comment]
Yes I agree; the idea is that it is dynamic so there won't be a huge query filling up endless amounts of memory to begin with. 

As for actually how to implement this, I'm still not 100% sure and it'll require some experimentation. 

So I'm going to push this back to 0.8.6 since 0.8.5 is already quite busy with more concrete things that can be implemented.

**Comment by zhyrohaad** [comment]
> .....so there won't be a huge query filling up endless amounts of memory to begin with.

taking memory on startup?  it wont...market seeding isnt controlled by server. (or shouldnt be)
simple query (that takes a while, yes), but can (and should) be executed outside the running (or not) server.
shouldnt affect available mem, and if it does, then host needs to upgrade.
even then, on live, market is it's own cluster.  i would love to add this functionality to evemu, but havent had time to iron bugs out, and have probably lost my notes and semi-working code along the years.
